README.rst

```diff
@@ -97,37 +96,38 @@ recently added or changed example experiment should be your best bet to
 get an accurate format. And if you find one that's out of date,
 submitting a pull request to fix it would be really nice!
 
+In order to get a feel for how the Deep QA package works, you should
+look at the `documentation <http://deep-qa.readthedocs.io/en/latest/index.html>`__.
+
 The best way currently to get an idea for what options are available in
 this configuration file, and what those options mean, is to look at the
 class mentioned in the ``solver_class`` field. Looking at the
-``dynamic_memory_network.json`` `(link) <https://github.com/allenai/deep_qa/blob/932849e8b3ebec6882680231924248669cc19758/example_experiments/dynamic_memory_network.json>`__
+``dynamic_memory_network.json`` `(link) <https://github.com/allenai/deep_qa/blob/master/example_experiments/dynamic_memory_network.json>`__
 example, we can see that it's using a
 ``MultipleTrueFalseMemoryNetworkSolver`` as it's
-``solver_class`` `(link) <https://github.com/allenai/deep_qa/blob/932849e8b3ebec6882680231924248669cc19758/example_experiments/dynamic_memory_network.json#L2>`__.
```

Comment:
We actually want these references to specific commits instead of master, because the line numbers in master may change (in fact have already changed since I wrote this).